src/figuresClasses.ts

```diff
@@ -1,19 +1,106 @@
+export type Shape = 'triangle' | 'circle' | 'rectangle';
+export type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
 
+  getArea(): number;
 }
 
-export class Triangle {
+export function isTriangleOk(a: number, b: number, c: number): boolean {
+  const longestSide = Math.max(a, b, c);
+  let isOk: boolean;
+
+  switch (longestSide) {
+    case a:
+      isOk = longestSide < (b + c);
+      break;
+
+    case b:
+      isOk = longestSide < (a + c);
+      break;
+
+    case c:
+      isOk = longestSide < (a + b);
+      break;
 
+    default:
+      isOk = false;
+      break;
+  }
+
+  return isOk;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape: Shape;
```

Comment:
```suggestion
  readonly shape = Shape.Triangle;
```
We should make shape readonly property as it is defined by class